bin/gw_summary

```diff
@@ -457,6 +458,18 @@ os.chdir(opts.output_dir)
 plotdir = os.path.join(path, 'plots')
 mkdir(plotdir)
 
+# add file containing a list of directories if required
+if (mode.get_mode().is_calendar() and
+        config.has_option("calendar", "highlight-available")):
+    # directory to list
+    dirpath = os.path.split(path)[0]
+    datedirs = sorted([thisdir for thisdir in os.listdir(dirpath)
+                       if os.path.isdir(os.path.join(dirpath, thisdir))])
+    dirfile = os.path.join(dirpath, 'list-dirs.txt')
+    config.set('calendar', 'date-file', dirfile)
```

Comment:
@mattpitkin, my one concern here is that users won't be able to pass a `date-file` in the INI configuration because it will be overwritten by the call to `config.set`. I think you can work around this pretty easily though, by changing from `set` to `get`:

```suggestion
    config.get('calendar', 'date-file', dirfile)
```

I haven't tested this though, so you'll probably want to do that.

Reply:
In 224cbefea02e0acf02d44cbe33de86d7125fcc35 I've made a change to check if the user is passing a date-file themselves.